apps/src/templates/EligibilityChecklist.jsx

```diff
@@ -0,0 +1,228 @@
+                      value="no"
+                      onChange={() => {this.handleYearChange(false);}}
+                      disabled={this.state.statusYear !== Status.UNKNOWN}
+                    />
+                    {i18n.eligibilityYearNo()}
+                  </label>
+                  <label>
+                    <input
+                      type="radio"
+                      name="year"
+                      value="yes1718"
+                      onChange={() => {this.handleYearChange(true);}}
+                      disabled={this.state.statusYear !== Status.UNKNOWN}
+                    />
+                    {i18n.eligibilityYearYes1718()}
+                  </label>
+                  <label>
+                    <input
+                      type="radio"
+                      name="year"
+                      value="yes1819"
+                      onChange={() => {this.handleYearChange(true);}}
+                      disabled={this.state.statusYear !== Status.UNKNOWN}
+                    />
+                    {i18n.eligibilityYearYes1819()}
+                  </label>
+                  <label>
+                    <input
+                      type="radio"
+                      name="year"
+                      value="yesAfter"
+                      onChange={() => {this.handleYearChange(false);}}
+                      disabled={this.state.statusYear !== Status.UNKNOWN}
+                    />
+                    {i18n.eligibilityYearAfter()}
+                  </label>
+                  <label>
+                    <input
+                      type="radio"
+                      name="year"
+                      value="unsure"
+                      onChange={() => {this.handleYearChange(false);}}
+                      disabled={this.state.statusYear !== Status.UNKNOWN}
+                    />
+                    {i18n.eligibilityYearUnknown()}
+                  </label>
+                  {/* Remove button after choice is made */}
+                  {this.state.statusYear === Status.UNKNOWN &&
+                    <Button
+                      color="orange"
+                      text={i18n.submit()}
+                      onClick={this.handleSubmit}
+                    />
+                  }
+                </form>
+              </div>
+            </div>
+          </div>
+        }
+        {this.state.statusYear === Status.FAILED &&
+          <div>{i18n.eligibilityYearDecline()}</div>
+        }
+        {this.state.statusYear === Status.SUCCEEDED &&
+          <div>
+            <SchoolAutocompleteDropdown
+              setField={this.handleDropdownChange}
+              value={submission.nces}
+              showErrorMsg={errors.nces}
+            />
+            {this.state.submission.nces !== "-1" && (
+              <Button
+                color="orange"
+                text={i18n.confirmSchool()}
+                onClick={() => {this.setState({displayDiscountAmount: true});}}
```

Comment:
FYI, when you have one-liner functions like this, you don't need the braces around it (or the semi-colon). This could just be:
```js
onClick={() => this.setState({displayDiscountAmount: true})}
```

That said, we should avoid creating functions in our render method, so I'd instead do something like
```
onClick={this.displayDiscountAmount}
```
and elsewhere:
```
displayDiscountAmount = () => {
  this.setState({displayDiscountAmount: true })
};
```